src/Client/Articles.php

```diff
@@ -117,7 +117,9 @@ public function slice(int $offset, int $length = null) : Sequence
                 $articles = [];
 
                 foreach ($result['items'] as $article) {
-                    if (isset($this->articles[$article['id']])) {
+                    if (isset($article['-invalid'])) {
+                        $articles[] = null;
```

Comment:
It is rather specific to the article content type, but the api-sdk is not accepting `null` as an input to its normalizers; unless the client modifies also the `count`, but that may break pagination too... Wanted to be as little invasive as possible